example/resources/users.py

```diff
@@ -71,3 +71,13 @@ def replace(self, params):
             'href': new_item['photos'].url()
         }
         return user
+
+
+@royal.collection_config('users.followers')
+class UserFollowers(royal.Collection):
+    pass
+
+
+@royal.item_config('users.followers')
+class Follower(Item):
+    pass
```

Comment:
Sure, example is quite confusing. Let me remove this ``Follower`` weird class and come up with a clearer example.